### PR TITLE
docs: fixes Readme #203

### PR DESCRIPTION
**Pull request by SteveLasker**

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Review by henkbirkholz — APPROVED** [review]

**Review by thomas-fossati — APPROVED** [review]
LGTM

Non blocking comments:

* Maybe add a full stop at the end of the sentence.
* Also, _"managed through github"_ seems superfluous.

File: README.md [hunk 1]
```diff
@@ -8,9 +8,7 @@ A golang library for the [COSE specification][cose-spec]
 
 ## Project Status
 
-**Current Supported Release**: [go-cose v1.1.0][current-release]
-
-**Current Pre-Release**: [go-cose v1.3.0-alpha.1][current-pre-release]
+The verasion/go-cose project is actively maintained. See [current releases](https://github.com/veraison/go-cose/releases) managed through github
```
Comment (SteveLasker) [inline README.md:11]:
```suggestion
The verasion/go-cose project is actively maintained.
See [current releases](https://github.com/veraison/go-cose/releases).
```

**Comment by SteveLasker** [comment]
Thanks, @thomas-fossati, I've made those changes.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**